### ISSUE TITLE
Comtypes module for Handy Tech HtBrailleDriverServer should be provided in comInterfaces

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#6546]
For the Handy Tech braille driver com server, the comtypes module is generated on the fly and stored in a temporary folder. Shouldn't this generated module be provided by default instead? I'd be happy to provide a pull request for this.

N.b. reported on behalf of @BabbageCom

**Comment by feerrenrut** [comment]
@jcsteh Would you mind commenting on this?

**Comment by jcsteh** [comment]
I initially wanted to keep drivers completely self-contained, but this change requires a change to our build system. This is why it wasn't done originally. However, we've since had to tweak our manifest for the Handy Tech driver anyway, so we've already made an exception and I guess I can live with another one. So, feel free to go ahead and implement this.

P4 because I don't think this has any noticeable impact, but we'll still take a PR.

**Comment by leonardder** [comment]
I've seen cases where python wrapper generation with comtypes resulted in an empty module, although not for this one.

**Comment by jcsteh** [comment]
All we do in the build system is use comtypes to generate the wrapper, so
if this does result in an empty module, it could break in our builds as
well, no?

**Comment by leonardder** [comment]
It could, but the chance that will happen is lower, only since it is 
generated only once per build instead of on every system

Sent on behalf of @BabbageCom

**Comment by jcsteh** [comment]
Right, except that if this happens, it'll break the driver for everyone,
not just one person. The take away is that this is not a reasonable excuse
to implement this change. If that bug occurs, we should by all means
address it, but that's a very separate issue.
